src/pages/en/guides/markdown-content.md

```diff
@@ -222,6 +222,42 @@ Each Markdown file exports the following properties:
 - `file`: The absolute path of this file (e.g. `/home/user/projects/.../file.md`).
 - `url`: If it's a page, URL of the page (e.g. `/en/guides/markdown-content`).
 - `getHeaders()`: An async function that returns the headers of the Markdown file. The response follows this type: `{ depth: number; slug: string; text: string }[]`.
+- `rawContent()`: A function that returns the raw content of the markdown file (excluding the frontmatter block) as a string. This is helpful when, say, calculating "minutes read." This example uses the [popular reading-time package](https://www.npmjs.com/package/reading-time):
+
+  ```astro
+  ---
+  import readingTime from 'reading-time';
+  const posts = await Astro.glob('./posts/**/*.md');
+  ---
+
+  {posts.map((post) => (
+    <Fragment>
+      <h2>{post.frontmatter.title}</h2>
+      <p>{readingTime(post.rawContent()).text}</p>
+    </Fragment>
+  ))}
+  ```
+
+- `rawContent.html()`: An asynchronous function that returns the raw content parsed to HTML. Note: **this does not parse `{jsx expressions}`, `<Components />` or layouts**! Only standard markdown blocks like `## headings` and `- lists` will be parsed. This is useful when, say, rendering a summary block for a blog post. This example plucks out the first paragraph as a summary using the [popular node-html-parser package](https://www.npmjs.com/package/node-html-parser):
```

Comment:
Maybe on it's own line ? And avoiding the repetition ("This is helpful when, say, ###" with "This is useful when, say, ###") from line 225
```suggestion
- `rawContent.html()`: An asynchronous function that returns the raw content parsed to HTML.

> **Note: This does not parse `{jsx expressions}`, `<Components />` or layouts**! Only standard Markdown blocks like `## headings` and `- lists` will be parsed. This is useful when rendering a summary block for a blog post. This example plucks out the first paragraph as a summary using the [popular node-html-parser package](https://www.npmjs.com/package/node-html-parser):
```

Reply:
@Chrissdroid I'm totally with you, I didn't like the "squishy-ness" either! 

BUT, with bullet points at the moment, it's probably best just to leave it, and not have multiple paragraphs and call outs within bullet points. 

I have made an issue that we're going to go through this entire page and re-format/space things out to not use bullet points anymore, and we can fix it at that time! So, I told Ben it was fine as is.  :)

(Good spot on the Markdown above, too! 🚀  )

Reply:
Updated suggestion ! 🚀 
```suggestion
- `rawContent.html()`: An asynchronous function that returns the raw content parsed to HTML. Note: **This does not parse `{jsx expressions}`, `<Components />` or layouts**! Only standard Markdown blocks like `## headings` and `- lists` will be parsed. This is useful when rendering a summary block for a blog post. This example plucks out the first paragraph as a summary using the [popular node-html-parser package](https://www.npmjs.com/package/node-html-parser):
```

Reply:
Agreed! These are fan-nit-tastic notes for @bholmesdev !

Reply:
I think we've agreed naturally on the docs on keeping uppercase format for libs and languages names like `React`, `Vue`, `Solid` or `JavaScript`, `TypeScript`, `Markdown`, etc...